public/main.js

```diff
@@ -538,24 +538,18 @@ const mapComp = Vue.component('map-comp', {
     data() {
         return {
             tooltip: null,
+            mapData: {},
+            loaded: false,
             // input: "0,0",
-            // debug: "",
-            // debug: "hide-input",
         }
     },
     mounted() {
-        if (this.$refs.tooltip) {
-            this.tooltip = Popper.createPopper(this.$refs.slot, this.$refs.tooltip, {
-                placement: 'right',
-                modifiers: [{ name: 'offset', options: { offset: [0, 5] } }],
+        fetch("https://map.runfast.stream/js/tilesources.json")
+            .then(res => res.json())
```

Comment:
A good habit here is something like:

```suggestion
            .then(res => (res.ok ? res.json() : Promise.reject(`HTTP ${res.status}: ${res.statusText}`)))
```

Fetch promises can resolve (successfully complete) with a failure (status in 4xx-5xx) or an unexpected result (status in the 1xx or 3xx range), so you've gotta be sure to catch both "successful failures" and failures